services/convert/issue_comment.go

```diff
@@ -66,6 +68,21 @@ func ToTimelineComment(ctx context.Context, c *issues_model.Comment, doer *user_
 		return nil
 	}
 
+	// for time tracking comments, we now store seconds
+	// so convert them for the API
+	if c.Content != "" {
+		if (c.Type == issues_model.CommentTypeAddTimeManual ||
+			c.Type == issues_model.CommentTypeStopTracking) &&
+			c.Content[0] == '|' {
+			i, _ := strconv.ParseInt(c.Content[1:], 10, 64)
+			c.Content = util.SecToTime(i)
+		} else if c.Type == issues_model.CommentTypeDeleteTimeManual &&
+			c.Content[0] != '-' {
+			i, _ := strconv.ParseInt(c.Content, 10, 64)
+			c.Content = "- " + util.SecToTime(i)
+		}
+	}
```

Comment:
Add some comments what these `|` and `-` prefixes mean? Do they ever show up in frontend rendered view? No, right?

Reply:
the dont show up ...

the `-` is the legacy of a finished rendered string for this type so i can identify legacy content that way and make seconds the default

for the other two types there is no final indicator if it's legacy ... so I did add the `|` as prefix to mark as new content format (seconds) instead of a finished formated string

we can not migrate the formated string back as this formation is looseful and each convertion can have some significant errors if you convert back and forth ... I tested that

Reply:
Ok, just add that info as a code comment please :)

Reply:
@silverwind the WebUI dont show any difference at all - the resulting html is the exact same

only now we will have the possibility for custom formations later

Reply:
I get that, but I just thought if someone comes across this, they may be confused by these obscure prefixes. BTW, how about "+" instead of "|" for added time?

Reply:
well it would also be a dellimeter - not sure I think that's bickeshading as nobody beside the database ever sees this

Reply:
@silverwind documented wherever there is that conversion or checking and on creation

Reply:
Thanks!